e2e/suites/management/create_new_scheduler_version_test.go

```diff
@@ -201,7 +211,7 @@ func TestCreateNewSchedulerVersion(t *testing.T) {
 			require.NoError(t, err)
 
 			// No version was created
-			require.Len(t, getVersionsResponse.Versions, 1)
+			require.Len(t, getVersionsResponse.Versions, 2)
```

Comment:
Since we have 2 now, a version was created
Maybe this test is showing a failure on maestro?